democracy/tests/test_hearing.py

```diff
@@ -233,6 +235,16 @@ def test_24_get_report(api_client, default_hearing):
 
 
 @pytest.mark.django_db
+def test_get_hearing_check_section_type(api_client, default_hearing):
+    response = api_client.get(get_hearing_detail_url(default_hearing.id))
+    data = get_data_from_response(response)
+    introduction = data['sections'][0]
+    assert introduction['type'] == InitialSectionType.INTRODUCTION
+    assert introduction['name_singular'] == 'johdanto'
```

Comment:
Hmm? Parts of the SectionType become properties of the Section in the serialization?